### ISSUE TITLE
simple-express-server installation issue - wrong address for clone ?

### ISSUE THREAD

**Issue by tfiloche** [5G-MAG/rt-common-shared#42]
Hi Daniel, @dsilhavy 

I follow the readme to install simple express server:
$ git clone --recurse-submodules https://github.com/5G-MAG/rt-mbms-examples
I was apparently  in "development" branch
I got some files and a "flute-ffmpeg" directory
I do not have a simple-express-server directory like indicated in the readme:

$ cd rt-mbms-examples/simple-express-server

I switch to main branch, and that the same situation

The instruction is maybe:
$ git clone --recurse-submodules https://github.com/5G-MAG/rt-common-shared

We have then a folder simple-express-server
Is it right ?

Then, the instruction after is:
npm install

Could you recommend a way to install npm in git bash for windows ?

**Comment by dsilhavy** [comment]
@tfiloche 

* We recently moved the `simple-express-server` to the rt-common-shared repository because it is used by multiple projects. I adjusted the getting started guide, but I might have missed some documentation. Which documentation did you use?
* I use `nvm` on Mac to manage multiple node versions. Can't you use the standard recommendations for Windows documented here: https://nodejs.org/en/download/package-manager
